src/main/java/net/dv8tion/jda/internal/entities/RoleImpl.java

```diff
@@ -60,7 +60,7 @@ public class RoleImpl implements Role, RoleMixin<RoleImpl>
     private boolean hoisted;
     private boolean mentionable;
     private long rawPermissions;
-    private int color;
+    private RoleColorsImpl colors;
```

Comment:
I would prefer storing this as 3 `int` fields instead, to reduce memory overhead. We can construct a `RoleColors` object in the getter.